extensions/amp-analytics/amp-video-analytics.md

```diff
@@ -0,0 +1,142 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-video-analytics"></a>AMP and video analytics
+
+
+## Video analytics triggers
+
+AMP video extensions issue various analytics events during their lifecycle. These events can be reported through the analytics configuration using `video-*` triggers.
+
+See [amp-analytics.md](../amp-analytics/amp-analytics.md) for details on *amp-analytics* configuration.
+
+
+### Video played trigger (`"on": "video-played"`)
+
+The `video-played` trigger is fired when the video begins playing from a user clicking play or from autoplay beginning or resuming. Use these configurations to fire a request for this event.
+
+```javascript
+"triggers": {
+  "myVideoPlayed": {
+    "on": "video-played",
+    "request": "event"
+  },
+  "videoSpec": {/* optional videoSpec */}
+}
+```
+
+### Video paused trigger (`"on": "video-paused"`)
+
+The `video-paused` trigger is fired when the video stops playing from a user clicking pause, from autoplay pausing, or from the video reaching the end. Use these configurations to fire a request for this event.
+
+```javascript
+"triggers": {
+  "myVideoPaused": {
+    "on": "video-paused",
+    "request": "event"
+  },
+  "videoSpec": {/* optional videoSpec */}
+}
+```
+
+### Video ended trigger (`"on": "video-ended"`)
+
+The `video-ended` trigger is fired when the video has reached the end of playback. Use these configurations to fire a request for this event.
+
+```javascript
+"triggers": {
+  "myVideoEnded": {
+    "on": "video-ended",
+    "request": "event"
+  },
+  "videoSpec": {/* optional videoSpec */}
+}
+```
+
+### Video session trigger (`"on": "video-session"`)
+
+The `video-session` trigger is fired when a "video session" has ended. A video session starts when a video is played and ends when video pauses, ends, or goes out of the viewport. This trigger is configurable in the `videoSpec` by setting `end-session-when-invisible`. Autoplaying videos can be optionally removed from this reporting by setting `exclude-autoplay` in the `videoSpec`. Use these configurations to fire a request for this event.
```

Comment:
`goes out of the viewport` wording is inconsistent with `end-session-when-invisible` and reader may not make the connection that they are talking about the same thing. 

I would reword as:

A video session starts when a video is played and ends when video pauses, ends or becomes invisible (optional configuration)

This trigger is configurable via the `videoSpec`. Configuration options include:

1- `end-session-when-invisible`: Will end the session when video becomes invisible by getting scrolled out of viewport.

2- `exclude-autoplay` : Will exclude autoplaying videos from this reporting.